_apps/com.github.vinser.pixyne.md

```diff
@@ -5,20 +5,21 @@ home:
 icon: https://github.com/vinser/pixyne/raw/master/appIcon.png
 
 screenshots:
-- url: https://github.com/vinser/pixyne/raw/master/docs/pixyneapp.jpg
-- url: https://github.com/vinser/pixyne/raw/master/docs/pixyneapp2.jpg
+- url: https://github.com/vinser/pixyne/raw/master/docs/pixyneapp.png
+- url: https://github.com/vinser/pixyne/raw/master/docs/pixyneapp2.png
 
-date:      2023-04-11 13:52:00
+date:      2023-11-14 21:10:00
```

Comment:
I don't think this should be updated. It is the date that the application was added to the page. Updating it will result in the app being shown as a new addition.

Reply:
Surely this date must stay as is in the case it used to show new arrivals. 
Should I delete this PR and make new one?

Reply:
We can add a new field which is "updated" then we are not erasing info on update.

Reply:
I don't think this was resolved.
The new date could be put in an "updated" field and we could use that for the requested #58 feature.